crates/json-comments-rs/src/lib.rs

```diff
@@ -0,0 +1,546 @@
+//! `json_comments` is a library to strip out comments from JSON-like test. By processing text
```

Comment:
The idea being that stuffing all of the crates into `packages/utils` is getting more and more confusing, so I would suggest putting all of the somewhat general purpose crates into crates itself (so dev-dep-resolver and node-resolver-rs). But you wanted to move some of them out of the monorepo anyway 🤷 
Otherwise I put this into `packages/utils` as well

Reply:
Yeah that makes sense.